zcash_client_sqlite/src/wallet.rs

```diff
@@ -232,10 +235,10 @@ impl ViewingKey {
         }
     }
```

Comment:
The `todo!()` on line 211 should be implemented. It should be straightforward given that you already have `UnifiedIncomingViewingKey::default_address(UnifiedAddressRequest)` in this PR:
```rust
    pub(crate) fn default_address(
        &self,
        request: UnifiedAddressRequest,
    ) -> Result<(UnifiedAddress, DiversifierIndex), AddressGenerationError> {
        match &self.viewing_key {
            ViewingKey::Full(ufvk) => ufvk.default_address(request),
            ViewingKey::Incoming(uivk) => uivk.default_address(request),
        }
    }
```
And then it might as well be public, I think.

Reply:
Quite right. That was one of the main points of this PR and I'm shocked I missed it. 
Maybe in the merge conflicts, I lost it.